lametro/views.py

```diff
@@ -840,6 +845,54 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class TagAnalyticsView(LoginRequiredMixin, View):
+    login_url = "/cms/"
+
+    def get(self, request):
+        if settings.HEROKU_APP_NAME:
+            # Send the task to a background worker
+
+            # For Heroku Platform dyno creation API reference, see:
+            # https://devcenter.heroku.com/articles/platform-api-reference#dyno-create
+            url = f"https://api.heroku.com/apps/{settings.HEROKU_APP_NAME}/dynos"
+            email = request.user.email
+            command = f"python manage.py generate_tag_analytics --email {email}"
+
+            data = {
+                "command": command,
+                "attach": False,  # Start a detached (daemon) dyno
+                "type": "run",
+            }
+            headers = {
+                "Accept": "application/vnd.heroku+json; version=3",
+                "Authorization": f"Bearer {settings.HEROKU_KEY}",
+            }
+
+            logger.info(f"Sending command to Heroku: {command}")
+            response = requests.post(url, json=data, headers=headers)
+            logger.info(
+                f"Got {response.status_code} response from Heroku: {response.json()}"
+            )
+
+            try:
+                response.raise_for_status()
+            except requests.exceptions.HTTPError:
+                messages.error(
+                    request,
+                    f"Received a {response.status_code} status code from Heroku. "
+                    "Analytics not generated.",
+                )
+            else:
+                messages.info(request, "Tag analytics are being generated...")
```

Comment:
It'd be nice to let the user know to expect an email containing a link to the generated file. We should also say that the file take some time to generate.

Reply:
Good point, I've got that in there now!